pennsignals/gentry
# Convert Consul Watch data to Slack message

Add converter to convert Consul Watch data to a strongly typed object ([`PostMessage`](https://github.com/pennsignals/gentry/blob/design/watch-data-converter/slack.go#L48)).

**Verification steps:**

Test using native go binary:

    $ go test -v ./...

Test using a Docker container:

    $ docker run --rm -v "$PWD":/usr/src/gentry -w /usr/src/gentry golang go test -v ./...

See #26
